src/bci_build/package/gcc.py

```diff
@@ -0,0 +1,88 @@
+"""Build description for the GCC Container Images"""
+
+from typing import Literal
+
+from bci_build.package import CAN_BE_LATEST_OS_VERSION
+from bci_build.package import DOCKERFILE_RUN
+from bci_build.package import LanguageStackContainer
+from bci_build.package import OsVersion
+from bci_build.package import Replacement
+
+
+def _is_latest_gcc(os_version: OsVersion, gcc_version: Literal[7, 12, 13, 14]) -> bool:
+    if os_version == OsVersion.TUMBLEWEED and gcc_version == 14:
+        return True
+    if os_version == OsVersion.SP6 and gcc_version == 13:
+        return True
+    if os_version == OsVersion.SP5 and gcc_version == 12:
```

Comment:
both sp5 and sp6 have gcc 13. the toolchain extension is not service pack agnostic so the version between should be always the same